### DISCUSSION TITLE
https://devko.me/%ED%94%84%EB%A1%A0%ED%8A%B8%EC%97%94%EB%93%9C/styled-components-container-query

### DISCUSSION THREAD

**Discussion by giscus** [seoseonyu/publish-quartz#17] [Announcements]
# https://devko.me/%ED%94%84%EB%A1%A0%ED%8A%B8%EC%97%94%EB%93%9C/styled-components-container-query

2024년 10월 15일 기준 styled-components최신버전인 6.1.13 버전에서도 Container Query를 공식적으로 지원 하지 않는다. 컨테이너 쿼리(Container Queries) 는 CSS의 강력한 기능 중 하나로, 요소의 스타일을 부모 컨테이너의 크기 또는 특정 속성을 기준으로 동적으로 조정할 수 있게 해줍니다.

https://devko.me/%ED%94%84%EB%A1%A0%ED%8A%B8%EC%97%94%EB%93%9C/styled-components-container-query

**Comment by seoseonyu** [comment]
하하
